src/zc/FileStorage/snapshotintime.py

```diff
@@ -70,11 +72,10 @@ def main(args=None):
             hour, minute, second = (map(int, time.split(':'))+[0,0])[:3]
         else:
             hour = minute = second = 0
-        stop = repr(
-            ZODB.TimeStamp.TimeStamp(year, month, day, hour, minute, second)
-            )
+        stop = repr(ZODB.TimeStamp.TimeStamp(year, month, day, hour, minute, second)).replace("'", "").decode("string_escape")
```

Comment:
Uh, please don't do this?

There's proper API to convert a TimeStamp to an 8-bytestring.  IIRC it's `.raw()`?  At least in the modern `persistent.timestamp.TimeStamp` implementation.  I think ZODB.TimeStamp is an alias to that one nowadays.